tests/foreman/cli/test_repository.py

```diff
@@ -228,8 +229,14 @@ def test_positive_create_with_default_download_policy(self):
 
         @Assert: YUM repository with a default download policy
         """
+        default_dl_policy = entities.Setting().search(
```

Comment:
Right, for ui to save some time while preparing environment we use nailgun which is much faster than create those entities via webui. CLI isn't that slow and in most cases there's no need to use api calls in cli tests. We're trying to avoid CLI calls in API tests and API calls in CLI tests correspondingly wherever it's possible (except the cases when some functionality is missing for specific interface).
Quick example why - when some API path is broken i'm expecting to see corresponding API tests failing and i'm definitely not ok with CLI tests failing because of that.